README.md

```diff
@@ -4,6 +4,11 @@ A [Visual Studio Code](https://code.visualstudio.com/) [extension](https://marke
 
 The Jupyter Extension includes the Jupyter Keymaps and the Jupyter Notebook Renderers extensions by default. The Jupyter Keymaps extension provides Jupyter-consistent keymaps and the Jupyter Notebook Renderers extension provides renderers for MIME types such as latex, plotly, vega, and the like. Both of these extensions can be disabled or uninstalled.
 
+| Link | Description |
+|------|-------------|
+| [File an issue](https://github.com/microsoft/vscode-jupyter/issues/new/choose) | For reporting problems and suggesting enhancements |
```

Comment:
`Help `-> `report issue` from VSCode is nice since it auto-populates version info. Would it work to squeeze that info in here? If that feels too noisy than this change is also fine.

Reply:
I think the `help-report issue` isn't that great as it misses a lot of the information that we need , python information , whether its debugging, interactive window or notebook or the like.